conjur-oss/templates/deployment.yaml

```diff
@@ -132,7 +136,7 @@ spec:
               name: {{ .Release.Name }}-conjur-database-url
               key: key
         - name: CONJUR_ACCOUNT
-          value: {{ .Values.account }}
+          value: {{ .Values.account.name }}
```

Comment:
Good question - I don't have an answer at the moment

Reply:
I'd prefer to leave the `CONJUR_ACCOUNT` environment variable being set
as part of Helm install, unconditionally. Two cases to consider:
- If the user doesn't end up actually enabling authn-k8s after the Conjur cluster
   comes up, then this is set unnecessarily, but "no harm, no foul" since this
   won't effect operation.
- If the user does eventually enable authn-k8s, then this will be already set
   up, so one less thing to worry about or forget in setting up authn-k8s.